chia/harvester/harvester_api.py

```diff
@@ -408,10 +409,10 @@ async def lookup_challenge(
                 else:
                     constants = self.harvester.constants
                     # after the phase-out, ignore v1 plots
+                    phase_out_epochs = num_phase_out_epochs(constants)
                     if (
                         new_challenge.last_tx_height
-                        >= constants.HARD_FORK2_HEIGHT
-                        + (1 << constants.PLOT_V1_PHASE_OUT_EPOCH_BITS) * constants.EPOCH_BLOCKS
+                        >= constants.HARD_FORK2_HEIGHT + phase_out_epochs * constants.EPOCH_BLOCKS
```

Comment:
this is similar to what we do inside is_v1_phased_out right ?
i think we should consolidate

Reply:
it's only similar. It's not obvious what could be factored out and be an improvement. Do you have something specific in mind?

Reply:
updated